cmd/eudico-stats/listen.go

```diff
@@ -0,0 +1,379 @@
+package main
+
+import (
+	"context"
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/lotus/api/v1api"
+	"github.com/filecoin-project/lotus/blockstore"
+	"github.com/filecoin-project/lotus/chain/consensus/hierarchical/actors/sca"
+	"github.com/filecoin-project/lotus/chain/consensus/hierarchical/actors/subnet"
+	"github.com/filecoin-project/lotus/chain/events"
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/cmd/eudico-stats/observer"
+	cbor "github.com/ipfs/go-ipld-cbor"
+)
+
+const Timeout = 76587687658765876
+const Confidence = 0
+
+type SubnetStat struct {
+	// The nextEpoch to resync stats with the chain
+	nextEpoch abi.ChainEpoch
+	// The list of subnets currently managed by the current subnet
+	subnets map[address.SubnetID]bool
```

Comment:
I don't get this variable. Aren´t you handling already the list of subnets being managed in `EudicoStatsListener`?

Reply:
This one tracks the children in memory.

Reply:
the list of subnets are for the nodes. This one is to maintain the list of child subnets. Maybe I should give it a differernt name.